server/config.go

```diff
@@ -299,8 +306,17 @@ type ScheduleConfig struct {
 	RegionScheduleLimit uint64 `toml:"region-schedule-limit,omitempty" json:"region-schedule-limit"`
 	// ReplicaScheduleLimit is the max coexist replica schedules.
 	ReplicaScheduleLimit uint64 `toml:"replica-schedule-limit,omitempty" json:"replica-schedule-limit"`
+	// Schedulers support for loding customized schedulers
+	Schedulers SchedulerConfigs `toml:"schedulers, omitempty" json:"schedulers"`
```

Comment:
`schedulers,omitempty`, should remove the blank space. 
Does JSON need `omiempty` too?

Reply:
I'm not quite sure, but I find that all the json in config.go without omiempty, so I don't add it either.